### PR TITLE
fix frontend dev for safari

### PR DESCRIPTION
**Pull request by shaunlebron**
## Description

Content Security Policy for `script-src` is `*:8080`, causing the errors below when loading http://localhost:3000 in Safari.  Maybe safari doesn’t like asterisks in the policy pattern anymore?  Setting it to `http://localhost:8080` made the errors go away.

    ```
    [Error] Refused to load http://localhost:8080/app/dist/runtime.hot.bundle.js?082aa3024750fd9ba8c7 because it does not appear in the script-src directive of the Content Security Policy.
    [Error] Refused to load http://localhost:8080/app/dist/vendor.hot.bundle.js?13591091791f90b83abf because it does not appear in the script-src directive of the Content Security Policy.
    [Error] Refused to load http://localhost:8080/app/dist/styles.hot.bundle.js?5c4237d32fedd3e1e105 because it does not appear in the script-src directive of the Content Security Policy.
    [Error] Refused to load http://localhost:8080/app/dist/app-main.hot.bundle.js?165fb99b5e07640afb2a because it does not appear in the script-src directive of the Content Security Policy.
    ```

## How to verify

Make sure http://localhost:3000 loads in Safari.

### CODE REVIEW

File: src/metabase/util.cljc [hunk 1]
```diff
@@ -706,7 +706,7 @@
       [;; strip out any current symbols
        [#"[^\d,.-]+"          ""]
        ;; now strip out any thousands separators
-       [#"(?<=\d)[,.](\d{3})" "$1"]
```
Comment (john-metabase) [inline src/metabase/util.cljc:709]: What was that lookbehind doing? I have a feeling it had some obscure purpose that an untested feature depends on. Maybe not though.

**Review by john-metabase — COMMENTED** [review]
Someone more qualified needs to comment on the removal of the regex lookbehind here but otherwise I support it. At least the `*` -> `localhost` is uncontroversial.

File: src/metabase/util.cljc (hunk 1 above)
Comment (shaunlebron) [reply src/metabase/util.cljc:709]: Hi @camsaul, can you check if this lookbehind is needed or not?  Tests still pass when it’s removed.  (I saw it was introduced in https://github.com/metabase/metabase/pull/11716)

Comment (camsaul) [reply src/metabase/util.cljc:709]: It does do something, try parsing something like `.001` with and without that lookbehind. Not sure if that's a realistic thing we need to parse or not tho. We should check if that's even something that can come back in a PG `currency` column or not. If it's not then we can go ahead and remove it

Comment (shaunlebron) [reply src/metabase/util.cljc]: I’m going to restore the lookbehind since it works fine in latest safari.

**Review by john-metabase — APPROVED** [review]
Assuming everything still works for Chrome/FF folks 👍

**Comment by alxnddr** [comment]
> @alxnddr is it safe to assume all frontend dev happens through the localhost domain?

By default yes, but people may have different configs locally, I'd double-check in #engineering

**Pull request merged**